newa/__init__.py

```diff
@@ -554,13 +567,29 @@ class Request(Cloneable, Serializable):
     def fetch_details(self) -> None:
         raise NotImplementedError
 
-    def generate_tf_exec_command(self) -> tuple[list[str], dict[str, str]]:
+    def generate_tf_exec_command(self, ctx: CLIContext) -> tuple[list[str], dict[str, str]]:
         environment: dict[str, str] = {
             'NO_COLOR': 'yes',
             }
         command: list[str] = [
             'testing-farm', 'request', '--no-wait',
             ]
+        rp_token = ctx.settings.rp_token
+        rp_url = ctx.settings.rp_url
+        if rp_token and rp_url:
+            command += [
+                '--tmt-environment',
+                f'TMT_PLUGIN_REPORT_REPORTPORTAL_TOKEN={rp_token}',
+                '--tmt-environment',
+                f'TMT_PLUGIN_REPORT_REPORTPORTAL_URL={rp_url}',
+                '--tmt-environment',
+                'TMT_PLUGIN_REPORT_REPORTPORTAL_PROJECT=baseosqe',
+                '--tmt-environment',
+                'TMT_PLUGIN_REPORT_REPORTPORTAL_LAUNCH=ksrot_newa',
```

Comment:
Would it make sense to read project and launch from configuration right away? It does look like a temporary, testing set of values, I get that.

Reply:
`rp_project` is already read from a config file, clearly I forgot to update it here.
Regarding `rp_launch`, I have decided to add it to the `Request` class and read its value from the recipe of or (if not defined) use recipe file basename as a launch name. I think it makes more sense from a user perspective, although we will most likely change it in the future anyway.
Both should be addressed now.